src/transaction/log_applier.c

```diff
@@ -7160,12 +7160,12 @@ la_print_log_arv_header (const char *database_name, LOG_ARV_HEADER * hdr, bool v
   DB_DATETIME datetime;
   char timebuf[1024];
 
-  tloc = hdr->db_creation;
+  tloc = hdr->vol_creation_time;
   db_localdatetime (&tloc, &datetime);
   db_datetime_to_string ((char *) timebuf, 1024, &datetime);
 
   printf ("%-30s : %s\n", "DB name ", database_name);
-  printf ("%-30s : %s (%ld)\n", "DB creation time ", timebuf, tloc);
+  printf ("%-30s : %s (%ld)\n", "Creation time ", timebuf, tloc);
```

Comment:
```suggestion
  printf ("%-30s : %s (%ld)\n", "DB creation time ", timebuf, tloc);
```

추가적으로 위에 남긴 의견과 동일한 의견드립니다.

1. db_creation_time과 volume_creation_time 모두 출력되는게 좋겠습니다.
2. 수정하신 후 실제로 출력해 보면서, 동작을 이해해 보도록 합시다.

Reply:
수정하였습니다.